beesdoo_easy_my_coop/models/coop.py

```diff
@@ -10,7 +10,7 @@ def get_eater_vals(self, partner, share_product_id):
         vals = {}
         eater = share_product_id.eater
 
-        if partner.is_company or partner.age < self._majority:
+        if partner.is_company or 0 < partner.age < self._majority:
```

Comment:
what about a child that is not yet 1 year old? i suggest to check whether `partner.birthdate_date` is truthy first.

```suggestion
        if partner.is_company or (partner.birthdate_date and partner.age < self._majority):
```

also, since you’re fixing things here, `_majority` is a “constant”, so it should be written in capitals. personally, i would write it outside of the class, since it is not a property of the `subscription.request` model.

Reply:
[age compute function](https://github.com/OCA/partner-contact/blob/12.0/partner_contact_birthdate/models/res_partner.py#L20), for reference.

Reply:
I do like your solution better 👍

Reply:
Agree with @huguesdk